.github/workflows/release.yaml

```diff
@@ -136,20 +151,21 @@ jobs:
         run: |
           cd registry
           mvn versions:set -DnewVersion=${{steps.metadata.outputs.snapshot-version}} -DgenerateBackupPoms=false -DprocessAllModules=true
+          sed -i  "s/version\:\s.*/version: \'master\'/g" .antora/antora.yml
       - name: Commit Snapshot Version ${{steps.metadata.outputs.snapshot-version}}
         run: |
           cd registry
           git add .
           git commit -m"Automated update to next Snapshot Version: ${{steps.metadata.outputs.snapshot-version}}"
           git push
-      # - name: Tweet About The Release
-      #   uses: ethomson/send-tweet-action@v1
-      #   with:
-      #     status: "${{steps.metadata.outputs.project-name}} version ${{steps.metadata.outputs.release-version}} is out!  Check out the release notes here: ${{ steps.create_release.outputs.html_url }}"
-      #     consumer-key: ${{ secrets.TWITTER_CONSUMER_API_KEY }}
-      #     consumer-secret: ${{ secrets.TWITTER_CONSUMER_API_SECRET }}
-      #     access-token: ${{ secrets.TWITTER_ACCESS_TOKEN }}
-      #     access-token-secret: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
+      - name: Generate Project Documentation for Website
+        run: |
+          echo "********** Building Registry Documentation using Antora **********"
+          cd docs-playbook
+          docker pull apicurio/apicurio-docs-builder:latest
+          docker run -v $(pwd):/apicurio-docs-playbook -it apicurio/apicurio-docs-builder:latest
```

Comment:
@EricWittmann Github Actions doesn't support interactive mode yet and bash cannot allocate a tty. Hence, It will fail with the error `the input device is not a TTY`. If we need an interactive terminal and aren't running in a terminal on Linux or macOS, we need to use a different command-line interface. PowerShell is reported to include this support on Windows.

The simplest solution would be to remove the `-it` option from the docker run command to make it non-interactive and remove the TTY

Reply:
Good catch - it does not need to be interactive.  I'll remove that.